src/main/scala/cosas/typeSets.scala

```diff
@@ -230,10 +230,6 @@ object typeSets {
 
     def toMap[K, V](implicit toMap: ToMap[S, K, V]): Map[K, V] = toMap(s)
 
-    def parseFrom[X](x: X)(implicit parser: S ParseFrom X): parser.Out = parser(s, x)
-
-    def serializeTo[X](implicit serializer: S SerializeTo X): X = serializer(s)
-
```

Comment:
it should be in something for `AnyTypeSet.of[Denotations]` but probably there's no usable way of expressing that.